sagemaker-training-compiler/tensorflow/single_gpu_single_node/hyper-parameter-tuning.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # Define the tunung job
</pre><p>Typo: <code>tunung</code> -&gt; <code>tuning</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>